pages/sites.tsx

```diff
@@ -3,10 +3,27 @@ import { EditIcon } from '~/components/icons';
 import { useState } from 'react';
 import SiteCardLink from '~/components/SiteCard';
 import { withSites } from '~/lib/utils';
+import { SiteList } from '~/lib/types';
+import useSWR from 'swr';
 
+const ParseSiteUUIDs = (data: SiteList): string[] => {
```

Comment:
Camel case, also change usage below.
```suggestion
const parseSiteUUIDs = (data: SiteList): string[] => {
```